site/content/installation.md

```diff
@@ -1,38 +1,40 @@
+To confirm Python and `pip` (a package management tool) are both installed correctly, open a command prompt or terminal and run the following in the command line:
+`python --version` and `pip --version`.
 
 ## Choosing a Code Editor  
 
-There are a number of different code editors or integrated development environments (IDEs) for Python. The most popular ones among users at Urban include the following: 
+There are a number of different code editors or integrated development environments (IDEs) for Python. We recommend using [Visual Studio Code](https://code.visualstudio.com/) (or VS Code), which can be installed  from the [official website](https://code.visualstudio.com/download). VS Code can be used for standard .py files, as well as for Quarto and IPython notebooks (.qmd and .ipynb files).
 
-* [Jupyter Notebook](https://jupyter.org/) comes installed with Anaconda, and is similar to RMarkdown. Jupyter notebooks are web-based applications that let you edit and run blocks of code along with markdown text, equations, and visualizations in a single notebook file. 
-* [Spyder](https://www.spyder-ide.org/) also comes installed with Anaconda and is most similar to RStudio for R. It offers editing, analysis, and debugging support for Python. 
-* [Visual Studio Code](https://code.visualstudio.com/) (or VS Code) can be installed through Anaconda, and includes support for debugging, syntax highlighting, code completion, and more. It also offers thousands of extensions (including [Python-specific extensions](https://code.visualstudio.com/docs/languages/python), a [Quarto extension](https://quarto.org/docs/tools/vscode.html), and an [AWS extension](https://aws.amazon.com/visualstudiocode/)).  
+VSCode includes support for debugging, syntax highlighting, code completion, and more. It also offers thousands of extensions (including [Python-specific extensions](https://code.visualstudio.com/docs/languages/python), a [Quarto extension](https://quarto.org/docs/tools/vscode.html), and an [AWS extension](https://aws.amazon.com/visualstudiocode/)).  
 
 ## Installing Packages 
 
-Anaconda comes with a package management system called `conda`. To install a package, open your command prompt and run 
-
-`conda install package-name`
+We recommend using the package management tool `pip` that comes with Base Python. To install a package, you can run `pip install package-name` from the command line.
 
 Note that this must be done from the command line and **not** the Python session (in contrast to package management in R).
 
-Base Python has its own package management tool called `pip`. If a package is not available with `conda install`, you can try running 
+If you are working on a computer that already has Anaconda installed (and predates the TOS changes), you can still use `conda-forge` (the open-source alternative to `conda`) to install packages. To do this, you can run:
+- `conda config --add channels conda-forge`
+- `conda install package-name`
```

Comment:
I haven't personally used conda-forge before so this is Copilot-assisted guidance. Can check on an Urban computer with Anaconda already installed.